### PR TITLE
#T04 Update all_auth templates

### PR DESCRIPTION
**Pull request by Jays-T**
## Description

This PR adds
> The Hackathon logo to each all_auth template
> Styling for each all_auth template keeping with the theme/styling of the site
> Replaces a text-shadow with greater font-weight 
> Adds hackathon logo to the landing page

## Screenshots

landing_page_plus_logo
landing_page_plus_logo_register
landing_page_plus_logo_reset_password

## Additional Information

> Additional updates on all_auth templates will be forthcoming, with certain inputs not having placeholder text (those currently have labels still showing).

> Due to a number of people adding various elements, some of the border-radius, padding, margin areas are inconsistent with each other. This will be refactored to be made consistent in further updates by adding global variables for each.

@stefdworschak  I'm not sure why each new branch I create still holds all of my previous commits. It is very odd, I've created the new branches from the master branch... so I'm not sure what is happening...

## Does this introduce a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: hackathon/templates/hackathon/hackathon_list.html [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {% extends "base.html" %}
+
```
Comment (stefdworschak) [inline hackathon/templates/hackathon/hackathon_list.html:2]: Can you remove the unneeded blank line?

File: home/templates/home/index.html [hunk 2]
```diff
@@ -5,73 +5,84 @@
...
-                                        
-                                            <h6 class="card-subtitle mb-2 text-muted"><span class="drk-txt-hkth">Start: </span>{{ hackathon.start_date }} - <span class="drk-txt-hkth">End: </span>{{ hackathon.end_date }}</h6>
-                                                <p class="card-text">{{ hackathon.description }}</p>
-                                            {% if hackathon.organiser %}
-                                            <p><span class="drk-txt-hkth">Organiser: </span><a href="#" class="p-blue card-link">{{ hackathon.organiser }}</a></p>
-                                            {% endif %}
-                                            <a href="#" class="btn btn-ci mr-3">Read More</a>
-                                        </div>
+                {% for hackathon in hackathons %}
+                    {% if hackathon.organisation == 'Code Institute' or hackathon.organisation == user.organisation %}
```
Comment (stefdworschak) [inline home/templates/home/index.html:12]: Maybe we can change this to not actually hard code `Code Institute`, but use the first organisation `hackathon.organisation.id == 1`?

File: static/css/style.css [hunk 3]
```diff
@@ -33,6 +33,15 @@ a:hover {
     text-decoration: none;
 }
 
+/* Global content border/shadow */
+.neu-shadow {
```
Comment (stefdworschak) [inline static/css/style.css:37]: what does `neu` stand for?

File: static/css/style.css [hunk 4]
```diff
@@ -164,16 +173,36 @@ footer ul li a {
...
+
+.all-auth-forms {
+    margin: auto;
+}
+
 /* Landing Page */
 
 .landing-page {
-    margin: 10px 10px 30px;
+
```
Comment (stefdworschak) [inline static/css/style.css:191]: Please remove unneeded blank line.

File: templates/allauth/account/login.html [hunk 5]
```diff
@@ -37,6 +37,7 @@ <h1>{% trans "Sign In" %}</h1>
             <a href="{{ signup_url }}">sign up</a> first.{% endblocktrans %}</p>
     {% endif %}
 
+
```
Comment (stefdworschak) [inline templates/allauth/account/login.html:40]: Please remove the unneeded empty line.

File: templates/allauth/base.html [hunk 6]
```diff
@@ -32,8 +32,12 @@
             </ul>
         </div>
 
-        {% block content %}
-        {% endblock %}
+            {% block allauth %}
+            {% endblock %}
+
+            {% block authhacklogo %}
+
```
Comment (stefdworschak) [inline templates/allauth/base.html:39]: You can delete this blank line as well.

File: templates/includes/messages.html [hunk 7]
```diff
@@ -1,9 +1,14 @@
...
+<link rel="stylesheet" href="{% static 'css/profile.css' %}" />
+{% endblock %}
+
 {% if messages %}
 <div>
     {% for message in messages %}
-        <h1>{{ message }}</h1>
-        <!-- TO DO : help wanted -->
-        <small>TO DO : templates > includes > messages.html (styles)</small>
+        <h3>{{ message }}</h3>
```
Comment (stefdworschak) [inline templates/includes/messages.html:11]: I would change the styling on this and use a boostrap alert class maybe. Also, `h3` seems a bit big.

**Review by stefdworschak — APPROVED** [review]
LGTM

@Jays-T Looks good to go. Just a few small comments from me. 👍

**Comment by stefdworschak** [comment]
Also, please resolve the merge conflict.

File: home/templates/home/index.html (hunk 2 above)
Comment (Jays-T) [reply home/templates/home/index.html:12]: Okay

File: static/css/style.css (hunk 3 above)
Comment (Jays-T) [reply static/css/style.css:37]: It basically stands for my lack of experience 😆  have changed it to .main-content-shadow

File: templates/includes/messages.html (hunk 7 above)
Comment (Jays-T) [reply templates/includes/messages.html:11]: Haven't yet gotten around to fully customizing the messages yet. Will probably look to setup specific alert classes depending on the type of message. For now have reduced size to <h5>.

**Comment by Jays-T** [comment]
Conflicts resolved :)

**Review by stefdworschak — APPROVED** [review]
LGTM

**Pull request merged**